frontend/src/pages/org/crawls-list.ts

```diff
@@ -272,11 +274,10 @@ export class CrawlsList extends LiteElement {
             <sl-icon name="search" slot="prefix"></sl-icon>
           </sl-input>
         </div>
-
-        <div class="col-span-12 md:col-span-4 flex items-center">
-          <div class="text-neutral-500 mr-2">${msg("View:")}</div>
+        <div class="grow flex items-center">
+          <div class="text-neutral-500 mr-2 ml-2">${msg("View:")}</div>
```

Comment:
- `grow` only applies to flexbox items, it won't work with `grid`
- nitpick: if the label left and right margin will reasonably always be the same value, you can use `mx-{n}` to handle both
```suggestion
        <div class="flex items-center">
          <div class="text-neutral-500 mx-2">${msg("View:")}</div>
```